pkg/printers/internalversion/describe.go

```diff
@@ -2478,6 +2478,26 @@ func describeNodeResource(nodeNonTerminatedPodsList *api.PodList, node *api.Node
 	w.Write(LEVEL_1, "%s (%d%%)\t%s (%d%%)\t%s (%d%%)\t%s (%d%%)\n",
 		cpuReqs.String(), int64(fractionCpuReqs), cpuLimits.String(), int64(fractionCpuLimits),
 		memoryReqs.String(), int64(fractionMemoryReqs), memoryLimits.String(), int64(fractionMemoryLimits))
+
+	oirResource := make([]string, 0, len(allocatable))
+	for resource := range allocatable {
+		if helper.IsOpaqueIntResourceName(resource) {
```

Comment:
Now that #48922 merged, could we update this to use the new `helper.IsExtendedResourceName` function? For now it includes OIRs, and when the OIR deprecation cycle is complete it will be updated. For context, extended resources are resources like `my-domain.com/anything` (any resouces outside of the `*kubernetes.io/` namespace.) Also please s/opaque/extended in the diff.